src/components/shared/Shared.js

```diff
@@ -0,0 +1,116 @@
+import instagramLogo from "../../assets/img/svg/instafram-logo.svg";
+import youtubeLogo from "../../assets/img/svg/youtube-logo.svg";
+import twitterLogo from "../../assets/img/svg/twitter-logo.svg";
+import facebookLogo from "../../assets/img/svg/facebook-logo.svg";
+import { Link } from "react-router-dom";
+import "./Shared.css";
+import { getSharedFolderSample, getSharedUserSample } from "../../api";
+import { useEffect, useState } from "react";
+import SharedMain from "./SharedMain";
+
+function Shared() {
+  const [userProfile, setUserProfile] = useState(null);
+  const [ownerProfile, setOwnerProfile] = useState(null);
+
+  const handleSharedUserProfile = async () => {
+    let result;
+
+    result = await getSharedUserSample();
+    if (!result) return;
+
+    const { email, profileImageSource } = await result;
```

Comment:
이미 위에서 `await getSharedUserSample();`을 통해 result 를 가져왔고, 없는 경우에 대한 처리를 한 상황에서 await 를 쓴 것은 의미가 없을 거 같아요
```suggestion
    const { email, profileImageSource } = result;
```

Reply:
넵 수정하겠습니다~